### ISSUE TITLE
findFlags not working with `key` query param

### ISSUE THREAD

**Issue by kweiberth** [checkr/flagr#187]
## Expected Behavior
The findFlags endpoint should return an array when a query param `key` is given, e.g. from GET /api/v1/flags/?key=aKey

## Current Behavior
Receiving an error message:

```
{
    "message": "cannot query all flags. Error 1064: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'key = ?))' at line 1"
}
```

## Steps to Reproduce (for bugs)

See above request URL for reproducible API call.

## Context

I'm attempting to fetch a flag by the flag's key.

## Your Environment

* Version used (`flagr version`): checkr/flagr:latest (might be 1.0.12 or later?)
* MySQL engine: 5.7.17

**Comment by zhouzhuojie** [comment]
Should be fixed now by #188

**Comment by kweiberth** [comment]
Awesome thank you 🙏
